pkg/kn/commands/revision/list_flags.go

```diff
@@ -55,7 +53,7 @@ func (f *RevisionListFlags) ToPrinter() (hprinters.ResourcePrinter, error) {
 	return p, nil
 }
 
-// AddFlags receives a *cobra.Command reference and binds
+// Flags receives a *cobra.Command reference and binds
```

Comment:
revert this change?

Reply:
oops, not sure how this came in. Will fix.